extensions/smallrye-fault-tolerance/deployment/src/main/java/io/quarkus/smallrye/faulttolerance/deployment/SmallRyeFaultToleranceProcessor.java

```diff
@@ -287,6 +250,49 @@ void processFaultToleranceAnnotations(SmallRyeFaultToleranceRecorder recorder,
 
         AnnotationStore annotationStore = validationPhase.getContext().get(BuildExtension.Key.ANNOTATION_STORE);
         IndexView index = beanArchiveIndexBuildItem.getIndex();
+
+        // Add reflective access to fallback methods
+        for (AnnotationInstance annotation : index.getAnnotations(DotNames.FALLBACK)) {
+            AnnotationValue fallbackMethodValue = annotation.value("fallbackMethod");
+            if (fallbackMethodValue == null) {
+                continue;
+            }
+            String fallbackMethod = fallbackMethodValue.asString();
+
+            System.out.println("Looking for fallback method: " + fallbackMethod);
```

Comment:
Should be removed I guess?

Reply:
Yes, I will switch this to draft once the CI completes since there seem to be other issues as well.

Reply:
Ah, OK, understood.